core/types/imports.go

```diff
@@ -60,6 +62,10 @@ var (
 	SlimAccountRLP       = ethtypes.SlimAccountRLP
 	FullAccount          = ethtypes.FullAccount
 	FullAccountRLP       = ethtypes.FullAccountRLP
+	CalcUncleHash        = ethtypes.CalcUncleHash
+	NewBlock             = ethtypes.NewBlock
+	NewBlockWithHeader   = ethtypes.NewBlockWithHeader
+	CopyHeader           = ethtypes.CopyHeader
```

Comment:
Is it ok if I sort these alphabetically? To have a consistent order between coreth and subnet-evm? Or I can do a separate PR that works as well.